From: Paulo Almeida <paulo.almeida@timeboxed.co.ao>
Date: Sun, 16 Aug 2015 12:58:45 +0100
Subject: [PATCH] Ensure auto_reconnect if first connect fails


diff --git a/src/riakc_pb_socket.erl b/src/riakc_pb_socket.erl
index 40e67b2..6605054 100644
--- a/src/riakc_pb_socket.erl
+++ b/src/riakc_pb_socket.erl
@@ -1261,6 +1261,7 @@ init([Address, Port, Options]) ->
         {error, Reason} when State#state.auto_reconnect /= true ->
             {stop, {tcp, Reason}};
         {error, _Reason} ->
+            erlang:send_after(State#state.reconnect_interval, self(), reconnect),
             {ok, State};
         Ok ->
             Ok
